mobile/src/modules/auth/components/AccountListItem.tsx

```diff
@@ -41,24 +44,38 @@ export class AccountListItem extends React.PureComponent<Props> {
     onPress(account);
   }
 
-  render () {
+  render() {
     const { accountRS = '', balanceNQT = '', type = '' } = this.props.account;
     const styles = this.getStyles();
+    const swipeBtns = [{
+      text: 'Delete',
+      backgroundColor: 'red',
+      underlayColor: 'rgba(0, 0, 0, 0.6)',
+      onPress: () => { this.props.onDelete(this.props.account); }
```

Comment:
we can create

```
handleDeleteAccount = () => this.props.onDelete(this.props.account);
```

and use it here

```
onPress: this.handleDeleteAccount,
```

Current implementataion creates new function on every render